### DISCUSSION TITLE
Community Discord thread - Call for Moderators.

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#4100] [Ideas]
Following up from the monthly contributor standup today.  Motion to create a community chat site.  Most agreed that it would be nice to set one up and Discord sounded popular, Sean agreed to be the owner of the server, but didn't want to be responsible for running it.  A few folks have experience creating a moderating community chat.  If you are interested in in being invited as a member or stepping up to me a moderator, please comment on this thread with how you would like to participate.

**Comment by markdav-is** [comment]
I'm willing to help moderate as well as be a member.

**Comment by leigh-pointer** [comment]
Count me in
________________________________

**Comment by vnetonline** [comment]
Count me in also ... happy to moderate

**Comment by GregStevenson** [comment]
I would participate, would not make a good moderator.

**Comment by mdmontesinos** [comment]
I believe a discord channel is an excellent idea to promote debate and share ideas. I'd definitely join!

**Comment by sbwalker** [comment]
Here is an invitation link for a new Discord server for Oqtane:

https://discord.gg/B8wnxbaM

I am happy to make some folks moderators once they join

**Comment by remesq** [comment]
Seems like the Discord link expired.

**Reply by leigh-pointer** [reply]
Try this 
https://discord.gg/teyUjNrt

**Comment by DiegelB** [comment]
Any fresh links?

**Reply by leigh-pointer** [reply]
https://discord.gg/BnPny88avK

**Reply by thabaum** [reply]
@leigh-pointer hoping this link still works and "never expires" I will add it to our docs and readme now.

**Reply by leigh-pointer** [reply]
@thabaum 
This link never expires
